autoware_launch/rviz/autoware.rviz

```diff
@@ -1614,7 +1594,11 @@ Visualization Manager:
                           Enabled: true
                           Name: ObstacleStop
                           Namespaces:
-                            {}
+                            detection_polygons: true
```

Comment:
@beyzanurkaya You wanna add only this line, right? If so, could you clean up the diffs that are not related to this PR?

Reply:
@satoshi-ota Yes you're right. I cleaned it.

Reply:
@beyzanurkaya Thanks. Now, merge ready?